src/index.ts

```diff
@@ -1529,6 +1535,44 @@ function determineDragOperation(effectAllowed:string, dropEffect:string):string
     return DROP_EFFECTS[DROP_EFFECT.NONE];
 }
 
+function onDelayTouchstart(evt: TouchEvent) {
+    const el = evt.target;
+
+    const heldItem = () => {
+        end.off();
+        cancel.off();
+        scroll.off();
+        onTouchstart(evt);
+    };
+
+    const onReleasedItem = () => {
+        end.off();
+        cancel.off();
+        scroll.off();
+        clearTimeout(timer);
+    };
+
+    const timer = setTimeout(heldItem, config.holdToDrag);
+
+    const end = onEvt(el, 'touchend', onReleasedItem, this);
+    const cancel = onEvt(el, 'touchcancel', onReleasedItem, this);
+    const scroll = onEvt(window, 'scroll', onReleasedItem, this);
+}
+
+function onEvt(el: EventTarget, event: string, handler: () => any, context: any) {
+    if (context) {
+        handler = handler.bind(context);
+    }
+
+    el.addEventListener(event, handler, supportsPassive ? { passive: false } : false);
```

Comment:
No need to actually specify `passive: false` here as the event handlers registered with this method are passive. Pls change to `el.addEventListener(event, handler);`

Reply:
Fixed